architecture/additional_concepts/authorization.adoc

```diff
@@ -378,33 +378,38 @@ metadata:
   name: privileged
 priority: null
 readOnlyRootFilesystem: false
-requiredDropCapabilities: null
-runAsUser: <3>
+requiredDropCapabilities: [] <5>
+runAsUser: <6>
   type: RunAsAny
-seLinuxContext: <4>
+seLinuxContext: <7>
   type: RunAsAny
 seccompProfiles:
 - '*'
-supplementalGroups: <5>
+supplementalGroups: <8>
   type: RunAsAny
-users: <6>
+users: <9>
 - system:serviceaccount:default:registry
 - system:serviceaccount:default:router
 - system:serviceaccount:openshift-infra:build-controller
 volumes:
 - '*'
 ----
 
-<1> The `*FSGroup*` strategy which dictates the allowable values for the
+<1> A list of capabilities that can be requested by a pod. An empty list means
+that none of capabilities can be requested while the special symbol `***`
+allows any capabilities
```

Comment:
@ahardin-rh The only change that is specific to 3.6 is the usage of `*` in `allowedCapabilities`: https://github.com/openshift/origin/pull/12875#issuecomment-299547514